### PR TITLE
Add continuous integration

### PR DESCRIPTION
**Pull request by TheZoker**
- Adds travis
- Adds ts-lint
- Adds jest
- Adds coveralls 
- Adds greenkeeper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Make it easier for developers to use [JSONForms](https://github.com/eclipsesourc
 ## Continuous Integration
 The JSONForms project is build and tested via [Travis](https://travis-ci.org/). Coverage is documented by [Coveralls](https://coveralls.io).
 
-Current status: [![Build Status](https://travis-ci.org/eclipsesource/jsonforms-tooling.svg?branch=master)](https://travis-ci.org/eclipsesource/jsonforms-tooling) [![Coverage Status](https://coveralls.io/repos/eclipsesource/jsonforms-tooling/badge.svg?branch=master&service=github)](https://coveralls.io/github/eclipsesource/jsonforms-tooling?branch=master)
```
Comment (edgarmueller) [inline README.md]: Why did you change the URLs?

File: vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
 // tslint:disable:no-var-requires
 // tslint:disable:no-require-imports
 // tslint:disable:no-use-before-declare
+// tslint:disable:no-shadowed-variable
```
Comment (edgarmueller) [inline vscode-extension/src/extension.ts]: Do we really need to disable this?

Comment (TheZoker) [reply vscode-extension/src/extension.ts]: Yes for now I need to disable this, since travis would fail. But with the refactor the functions will be renamed anyway, so I'll remove it again in the refactor branch

File: README.md (hunk 1 above)
Comment (TheZoker) [reply README.md]: Actually a good question. I undid this change

File: vscode-extension/src/extension.ts (hunk 2 above)
Comment (edgarmueller) [reply vscode-extension/src/extension.ts]: Alright.
